cmd/pilot-agent/main.go

```diff
@@ -111,9 +115,13 @@ func init() {
 		"Sidecar proxy unique ID. If not provided uses ${POD_NAME}.${POD_NAMESPACE} environment variables")
 	proxyCmd.PersistentFlags().StringVar(&sidecar.Domain, "domain", "cluster.local",
 		"DNS domain suffix")
+	proxyCmd.PersistentFlags().StringVar(&customConfig, "envoyconfig", "",
+		"Custom envoy config")
 
 	cmd.AddFlags(rootCmd)
 
+	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
```

Comment:
I'm not sure what you mean - AddGoFlagSet means that any go flag defined using standard go will work, as expected. There is no reason to have to add them manually - or disable them.